src/app/dataset-view/additional-components/metadata-component/components/form-components/schema-field/schema-field.component.html

```diff
@@ -1,5 +1,6 @@
 <div class="d-flex flex-column dynamic-table" [formGroup]="form">
     <label class="form-label">{{ label }}</label>
+
     <ng-container [formArrayName]="controlName">
```

Comment:
Please work more with the validators:
- names of the columns should not repeat
- name cannot be the same as type
- name and type are required

Reply:
All validators was added.